docs/reference/indices/get-index.asciidoc

```diff
@@ -15,20 +15,3 @@ alias or wildcard expression is required.
 
 The get index API can also be applied to more than one index, or on
 all indices by using `_all` or `*` as index.
-
-[float]
-=== Filtering index information
-
-The information returned by the get API can be filtered to include only specific features
-by specifying a comma delimited list of features in the URL:
-
-[source,js]
---------------------------------------------------
-GET twitter/_settings,_mappings
```

Comment:
I think we could, but not sure it's worth it since I think this is a very minor edge case that people use.